app/views/schools/transfer_out/check_transfer.html.erb

```diff
@@ -9,16 +9,15 @@
     <span class="govuk-caption-xl"><%= @school.name %></span>
     <h1 class="govuk-heading-xl"><%= title %></h1>
 
-    <p class="govuk-body">If they’re leaving your school for a different reason, you must contact your training provider
-      to remove them instead. Reasons may include:</p>
-    <ul class="govuk-list govuk-list--bullet">
-      <li>leaving teaching</li>
-      <li>taking a break from teaching</li>
-      <li>taking a teaching job outside of England</li>
-    </ul>
-    <p class="govuk-body govuk-!-margin-bottom-7">If this person is moving to work at a new school where they’ll
-      continue their induction, confirm below.</p>
+    <p class="govuk-body">Confirm if this participant is moving to another school to continue their ECF-based training.</p>
 
-    <%= govuk_button_link_to "Continue",  teacher_end_date_school_participant_transfer_out_path %>
+    <div class="govuk-inset-text">
+      <p class="govuk-body">
+        If they are leaving your school for any other reason, do not confirm.<br/>
+        Contact your training provider instead.
+      </p>
+    </div>
```

Comment:
I don’t think we should use inset text here, as this is just a continuation of the guidance. It might seem surprising, but in all the usability tests I've observed, inset text often gets ignored more than regular text. 🤦 

We should also avoid single line breaks too.

```suggestion
    <p class="govuk-body">
      If they are leaving your school for any other reason, contact your training provider instead.
    </p>
```

Reply:
@frankieroberto yes I agree with your suggestion however, this is required by the design.